File: client/src/app/models/group-member.ts
import {User} from '@models/user';

export class GroupMember {
  id: number;
  userId: number;
  groupId: number;
  role: string;

  user?: User;

  constructor(id: number, userId: number, groupId: number, role: string) {
    this.id = id;
    this.userId = userId;
    this.groupId = groupId;
    this.role = role;
  }
}

File: server/WebApplication1/Models/GroupMember.cs
namespace WebApplication1.Models
{
    public class GroupMember
    {
        public int Id { get; set; }
        public int user_id { get; set; }
        public int group_id { get; set; }
        public string role { get; set; }
        public DateTime? created_at { get; set; }
    }
}
